Translated Descriptions

DahliaSR opened the pull request:
Closes #1

DahliaSR committed:
Translated Descriptions

Closes #1

GiForce commented on translations/de/info/task_description.html:
```diff
@@ -3,45 +3,76 @@
+    Das Labyrinth wird als Matrix (eine Liste von Listen) dargestellt: 1 ist ein Busch und
+    0 ist Teil des Weges.<br>
+    Die Größe des Labyrinths beträgt 12 x 12 und die äußeren Zellen sind ebenfalls Büsche.<br>
+    Der Spieler beginnt in der Zelle (1; 1). Der Ausgang ist in der Zelle (10; 10).<br>
```
Komma, nicht Strichpunkt ->Zelle (1**,** 1) Zelle (10**,** 10)

GiForce reviewed (changes requested):
👻

DahliaSR committed:
Fixes according to review

https://github.com/CheckIO-to-German/Open-Labyrinth/pull/3#pullrequestreview-5784652

GiForce reviewed (approved):
👾
